match/README.md

```diff
@@ -358,6 +358,19 @@ If you want to manually decrypt a file you can.
 openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d
 ```
 
+### <a name="import_certificate"></a>Import an existing certificate
+
+To quickly and easily get started with `match`, it is advised to revoke your existing certificates. Although, if revoking code signing certificates is not an available option for you, match provides an `import` feature that lets you import an existing code signing identity certificate, to your match's repository.
+
+Before using this feature it is required that the code signing certificate you want to import is properly installed in your Keychain and it is associated with an active code signing identity registered in your Apple Developer account. Check this [fastlane.docs page](https://docs.fastlane.tools/codesigning/xcode-project/) to easily speed up with this requirements.
+
+To import your code signing certificate, P12 encoded, run:
+
+```
+match import "<path-to-p12-code-signing-certificate>" "<p12-password-if-not-empty-string>"
```

Comment:
@KrauseFx fixed, thanks :)
let me know, what can I do to get this finally merged? ;)